pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.black]
+line-length = 80
+target-version = ['py34', 'py35', 'py36', 'py37', 'py38']
```

Comment:
Actually we should remove now target-versions py34 and py35 from all packages.

Reply:
Done